app/javascript/src/components/Common/FormPane.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import { Check } from "@bigbinary/neeto-icons";
+import { Formik, Form } from "formik";
+import { Pane, Typography, Button, Toastr } from "neetoui/v2";
+
+const FormPane = ({
+  entity,
+  isOpen,
+  close,
+  children,
+  initialValues,
+  validationSchema,
+}) => {
+  const submit = () => {
+    Toastr.success(`${entity} created successfully.`);
+    close();
+  };
+
+  return (
+    <Pane isOpen={isOpen} onClose={close}>
+      <Pane.Header>
+        <Typography style="h2" weight="semibold" className="px-2">
+          Add New {entity}
+        </Typography>
+      </Pane.Header>
+
+      <Formik
+        initialValues={initialValues}
+        validationSchema={validationSchema}
+        onSubmit={submit}
+      >
+        {() => (
```

Comment:
Get the `isSubmitting` formik boolean prop and use it to disable the submit button once it's clicked. Otherwise user can click on the submit more than once simultaneously.